src/Middlewares/InputFieldHandlerInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TheCodingMachine\GraphQLite\Middlewares;
+
+use TheCodingMachine\GraphQLite\InputField;
+use TheCodingMachine\GraphQLite\InputFieldDescriptor;
+
+/**
+ * @unstable See https://graphqlite.thecodingmachine.io/docs/semver.html
```

Comment:
What is meant by this annotation?  Link is broken.

Reply:
copied from FieldHandlerInterface

Reply:
Let's remove it then.  The link is dead and doesn't make sense.